kedro_neptune/__init__.py

```diff
@@ -138,46 +137,30 @@ def _load(self) -> neptune.run.Handler:
         return run[base_namespace]
 
 
-class AbstractNeptuneDataSet(AbstractDataSet, ABC):
-    """Abstract class for extending other DataSets.
-     Instances are produced by NeptuneArtifactDataSet factory."""
+def load_neptune_artifact(data_set: AbstractVersionedDataSet):
+    # pylint: disable=protected-access
+    load_path = get_filepath_str(data_set._get_load_path(), data_set._protocol)
 
-    def load_raw(self) -> Any:
-        pass
+    with data_set._fs.open(load_path, **data_set._fs_open_args_load) as fs_file:
+        return fs_file.read()
 
 
 class NeptuneArtifactDataSet(AbstractDataSet):
```

Comment:
does it make sense to keep this class after changes? maybe a factory function?

Reply:
I can't find a usage example in https://github.com/neptune-ai/kedro-neptune/pull/15/files , can you help me with that?